exporters/otlp/internal/envconfig/envconfig.go

```diff
@@ -91,18 +102,38 @@ func WithURL(n string, fn func(*url.URL)) func(e *EnvOptionsReader) {
 }
 
 // WithTLSConfig retrieves the specified config and passes it to ConfigFn as a crypto/tls.Config.
```

Comment:
Be sure to update the godoc if you're going to change this function name.

Reply:
done